modules/cluster-logging-clo-status-comp.adoc

```diff
@@ -50,8 +54,15 @@ Events:
 
 .. Get the name of a ReplicaSet:
 +
+.Example output
```

Comment:
This doesn't need `.Example output`.

Reply:
Why not? The output is going to be different every time.